Netcracker/KubeMarine
# [MANOPD-82359] Calico toleration fix

### Description
* Tolerations were previously used in templates and should processed in python code
* The Jinja2 templates for `Calico` should be deleted


### Solution
* Cover `tolerations` option in python code


### How to apply
Not applicable


### Test Cases

**TestCase 1**
Check if default toleration keys are processed

Test Configuration:

- Hardware: 4CPU/4GB
- OS: Ubuntu 22.04
- Inventory: AllInOne

Steps:

1. Run installation procedure
2. Check `tolerations` in `calico-typha` `Deploymet`

Results:

| Before | After |
| ------ | ------ |
| Tolerations from original YAML are presented | Tolerations both from original YAML and default are presented |


**TestCase 2**
Check if custom toleration keys are processed

Test Configuration:

- Hardware: 4CPU/4GB
- OS: Ubuntu 22.04
- Inventory: AllInOne

Steps:

1. Run installation procedure with custom `plugins.calico.typha.tolerations` in `cluster.yaml`
2. Check `tolerations` in `calico-typha` `Deploymet`

Results:

| Before | After |
| ------ | ------ |
| Toleration keys from original YAML are presented | Toleration keys includes the original YAML keys, the default keys, and the custom keys |


### Checklist
